### PR TITLE
Add CI Testing

### PR DESCRIPTION
**Pull request by jonathangreen**
Ticket:
Islandora-CLAW/CLAW#794

Doesn't test cent, but we should once cent is supported.

Currently fails, and will until https://github.com/Islandora-Devops/ansible-role-tomcat8/pull/6 is merged.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+services: docker
+
+env:
+  - distro: centos7
```
Comment (dannylamb) [inline .travis.yml]: Looks like it is trying to test centos? https://travis-ci.org/Islandora-Devops/ansible-role-blazegraph/jobs/338288091#L443

Comment (jonathangreen) [reply .travis.yml]: good call. missed that.

**Comment by jonathangreen** [comment]
Something more going on here then I thought... I'll take a look into it when a get a chance today.

**Comment by jonathangreen** [comment]
A big part of my issues here was making sure the Blazegraph configuration files are found correctly. I added a second way to specify it, which puts it in the Blazegraph web.xml. 

Its a little ugly to just template the web.xml file, and if anyone has better ideas let me know. I made it so you can switch it off and also put it in the java_opts if you want, but that requires the help of whatever role sets the java_opts variables.

I made some notes in the readme about all of this.

**Review by whikloj — APPROVED** [review]
👍

**Pull request merged**